api_tests/run-federation-test.sh

```diff
@@ -13,7 +13,7 @@ popd
 yarn
 yarn api-test || true
 
-killall lemmy_server
+killall -s1 lemmy_server
```

Comment:
Without this there were still lemmy processes left running after tests completed.

Reply:
ok, will start on this